chips/cc26x2/src/radio/commands.rs

```diff
@@ -37,25 +37,8 @@ bitfield! {
     impl Debug;
     pub _frontend_mode, set_frontend_mode: 2, 0;
     pub _bias_mode, set_bias_mode: 3;
-    pub _analog_cfg_mode, _set_analog_config_mode: 9, 4;
-    pub _no_fs_powerup, _set_no_fs_powerup: 10;
-}
-
-// Radio Command Operation Status
-
-#[derive(Debug, Clone, Copy)]
-pub enum RfcOperationStatus {
-    Idle,
-    Pending,
-    Active,
-    Skipped,
-    SendDone,
-    CommandDone,
-    LastCommandDone,
-    RxOk,
-    TxDone,
-    Setup,
-    Invalid,
+    pub _analog_cfg_mode, set_analog_config_mode: 9, 4;
+    pub _no_fs_powerup, set_no_fs_powerup: 10;
```

Comment:
what even is this syntax?

Reply:
Bitfield syntax. It’s a macro invocation like the registers macro but I have done this a bit backwards. Doesn’t affect its functionality but it would read better if you start at MSB and assign the fields down to LSB instead of LSB to MSB.